### ISSUE TITLE
Update solr 6 to 6.6.5

### ISSUE THREAD

**Issue by sepal** [docker-solr/docker-solr#184]
Solr 6.6.5 was released on the 3rd of July and while it doesn't comes out of the changelog, this is actually a security release. The issue [SOLR-12450](https://issues.apache.org/jira/browse/SOLR-12450) linked in the [change log](https://lucene.apache.org/solr/6_6_5/changes/Changes.html#v6.6.5.bug_fixes) points to the security issue with the number CVE-2018-8026, see also https://nvd.nist.gov/vuln/detail/CVE-2018-8026.

Therefore it would be really good to have the 6.6.5 release on docker hub.

**Comment by makuk66** [comment]
Thanks @sepal. I've just returned from several weeks' absence. I'll get that new version added.

**Comment by makuk66** [comment]
I saw that and I've merged that in. In case you're not aware: running `./tools/update.sh` will create that change for you automatically; and indeed running that produces no changes now, so you did it right :-)
I have a build failure to fix (https://www.travis-ci.org/docker-solr/docker-solr); I know what's going on there, but fixing that is gonna take a little time. Then going through the process in [update.md](https://github.com/docker-solr/docker-solr/blob/master/update.md). Aiming for Monday.
